README.md

```diff
@@ -57,6 +57,8 @@ Utilities are included in this repository to convert time-series data generated
 
 For loading of datasets to work correctly, you'll need to run these commands from a device that has access to Allen Institute's on-premises data storage. If running off of shared resources, remember to initialize your virtual environment first! This may look like `conda activate {my_env}`.
 
+NOTE: You will also need to run Python version 3.8 due to the project's dependencies. For conda, you can create an environment with `conda create --name {my_env} python=3.8`.
```

Comment:
I think we can manage this dependency differently.  It really only matters for generate_data.py and not for data_writer_utils or generate_data_2.  I'm not sure there's anything to change here except maybe to note exactly when/why you'd need 3.8.

Reply:
The issue happens when you install the `pyproject.toml`, which means that the EMT Migration script is also affected.  I guess they could be moved into separate folders with their own dependency files...?

Reply:
In a way, the actual packageable library is just data_writer_utils now, which could have a `>=3.8` requirement, and then the convert scripts are just separate.